frontend/src/ui-kit/components/CopyButton/CopyButton.tsx

```
@@ -0,0 +1,164 @@
+          disabled:cursor-not-allowed disabled:opacity-50
+          ${className}
+        `.trim()}
+      >
+        {renderIcon()}
+        {visualLabel && (
+          <span className="text-xs font-medium">{visualLabel}</span>
+        )}
+      </button>
+      {/* Accessible live region for status announcements */}
+      <span
+        role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)